package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "start": "node_modules/.bin/webpack-dev-server --inline --hot",
+    "eslint": "eslint ./scripts",
```

Comment:
We may need the path `node_modules/.bin/` in the command since eslint is not globally installed in CI environment.